test/connectivity/modules/NetworkController/docker/linux/arm32v7/base/Dockerfile

```diff
@@ -1,4 +0,0 @@
-ARG base_tag=2.1.13-bionic-arm32v7
```

Comment:
Refactored module base images.

Simple modules use module-base image. Complex modules use module-base-full image (contains aspnet core, rocksdb, traffic shaping).

We used to split this by exact tooling, and thus had 3 different module base images. When we then needed to add aspnetcore, rather than create another base image, I chose to consolidate them to reduce organizational complexity.